README.md

```diff
@@ -1,41 +1,112 @@
+
+### Method.
+
+The algorithm applied finds the least weight of a path in a directed, weighted “metagraph” in which each vertex represents a subset of vertices in the primal graph *G* (the one whose zero forcing number is desired) with the property that, when this subset is precisely the set of filled vertices, no vertex can force.  An arc of weight *w* is present from *X* to *Y* when it is possible to add *w* vertices to the initially filled set that produced *X* to obtain an initially filled set that produces *Y*.  (That is, it is possible to expand the size of the ultimately filled set from *X* to *Y* at the “cost” of filling *w* additional vertices at the beginning.)  Then one can show that the smallest total weight of a directed path from &empty; to *V(G)* in this metagraph is the zero forcing number of *G*.
+
+The advantage to the above formalism is that it allows the zero forcing number to be computed using any of the standard algorithms for finding a path of smallest weight in a weighted directed graph.  In particular, in this implementation we use Dijkstra's Algorithm.
+
+### Capabilities and limitations.
+
+The software is able to compute the zero forcing number very efficiently for most simple graphs.  Some graphs, such as stars, represent a weakness for the algorithm and may produce longer running times.  In addition, memory usage can be prodigious for very large graphs.  Steps are planned to address both of these limitations in the future.  For now, the algorithm is very quick for most graphs.  For example, for the Paley graph on 101 vertices, the zero forcing number can be computed in just a few seconds.
 
 ---
 
-## Install from PyPI
-```bash
-sage -pip install -i https://test.pypi.org/simple/ zeroforcing
+
+### How to install:
+
+#### Option 1. Install through PyPI
+
+Type the line below into any cell in a SageMath Jupyter notebook:
+
+```
+%pip install --index-url https://test.pypi.org/simple/ zeroforcing==0.1.0
 ```
 
-## Install from source
-```bash
-sage -pip install .
+Alternatively, type the following at the command line in any terminal session:
+
+```
+sage -pip install --index-url https://test.pypi.org/simple/ zeroforcing==0.1.0
 ```
 
-## Test
-### Install the dependencies for testing
-```bash
-sage -pip install zeroforcing[test]
+
+The code should download from PyPI and build using Cython. (If running on macOS, during the process you may be prompted to install the command line developer tools.  This is because installation involves compiling Cython code.)
+
+#### Option 2: Install from source
+
+Either use `git` to clone the repository into the directory in which your SageMath Jupyter notebook resides, or follow these steps:
+
+* Download the `.zip` file of the project from [the releases page](https://github.com/alexhutman/ZeroForcingNumber/releases).
```

Comment:
I'm wondering if we should just force people to clone from git. Building/uploading `zip`s for each version seems like it will become a huge pain in the ass, for little reward. We can probably make use of git tags to tag versions that users can checkout. Thoughts?

Reply:
Do we have to build and upload a `zip`?  I thought GitHub just automagically created one, i.e., that's where that file came from (that's at the URL).  But I agree if we have to actually create the file to post there, then forget it.  What is the situation with that?

Reply:
Oh yeah you might be right! We can maybe table this for now then -- what we have here is fine. We can revisit it if need be, shouldn't be too big of a deal!